src/soc/ibm/power9/istep_14_2.c

```diff
@@ -0,0 +1,51 @@
+#include <cpu/power/istep_14.h>
+#include <cpu/power/istep_13.h>
+
+static void thermalInit(void)
+{
+	for (size_t MCSIndex = 0; MCSIndex < MCS_PER_PROC; ++MCSIndex) {
+		for (size_t MCAIndex = 0; MCAIndex < MCA_PER_MCS; ++MCAIndex) {
+				mca_and_or(
+					mcs_to_nest[mcs_ids[MCSIndex]], MCAIndex, MCA_MBA_FARB3Q,
+					~PPC_BITMASK(0, 45),
+					PPC_BIT(10) | PPC_BIT(25) | PPC_BIT(37));
+		}
+		scom_and_for_chiplet(mcs_ids[MCSIndex], MCS_MCMODE0, PPC_BIT(21));
```

Comment:
```suggestion
		scom_and_for_chiplet(mcs_to_nest[mcs_ids[MCSIndex]], MCS_MCMODE0 + 0x80 * MCSIndex, PPC_BIT(21));
```
This is Nest register. Also, I am _guessing_ that it needs to be shifted by 0x80 for second MCS, this register is in the same range as https://github.com/3mdeb/coreboot/commit/92f34cc8e119c0a1af83a0304de2fe8df8d2eb9c. Without proper documentation there is no way of knowing...